test/support/managing_two_sv_helpers.rb

```diff
@@ -19,7 +19,7 @@ def assert_user_access_log_contains_messages(user, messages)
   end
 
   def mandate_2sv_for_exempted_user
```

Comment:
Minor: this method name could change to `turn_on_2sv_for_exempted_user` to match the text.

Reply:
I'd prefer to leave this method name unchanged. I think "mandate" is a more accurate description of what this is doing and the code doesn't always have to mirror the copy exactly.

Reply:
👍